docs/devel/scheduler_algorithm.md

```diff
@@ -47,6 +47,8 @@ The purpose of filtering the nodes is to filter out the nodes that do not meet c
 - `PodFitsHost`: Filter out all nodes except the one specified in the PodSpec's NodeName field.
 - `PodSelectorMatches`: Check if the labels of the node match the labels specified in the Pod's `nodeSelector` field ([Here](../user-guide/node-selection/) is an example of how to use `nodeSelector` field).
 - `CheckNodeLabelPresence`: Check if all the specified labels exist on a node or not, regardless of the value.
+- `MaxEBSVolumeCount`: Ensure that the number of attached ElasticBlockStore volumes does not exceed a maximum value (by default, 39, since Amazon recommends a maximum of 40 -- see [Amazon's documentation](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits)).  The maximum value can be controlled by setting the `KUBE_MAX_PD_VOLS` environment variable.
```

Comment:
Eventually there will likely be at least one cloud (AWS) that has at more than one PD type (EBS and EFS), so I would avoid giving names like `KUBE_MAX_PD_VOLS`.  How about `MAX_ATTACHED_GCEPD_VOLS` and so forth?